src/clients/db/PageRequest.ts

```diff
@@ -0,0 +1,13 @@
+export default interface PageRequest {
+  number: number | undefined
+  size: number | undefined
+  nextPageInfo: { [key: string]: string | null } | undefined
+  previousPageInfo: { [key: string]: string | null } | undefined
+  direction?: 'previous' | 'next'
+}
+export const UnpagedRequest: PageRequest = {
```

Comment:
real nitpick here: there should be a new line between 7 & 8

Reply:
Done